openstf/pipeline/predict_sklearn.py

```diff
@@ -1,18 +1,19 @@
-from datetime import datetime, timedelta
+from datetime import datetime, timedelta, timezone
 from pathlib import Path
 
 import joblib
 import pandas as pd
-
+import structlog
 
 from ktpbase.database import DataBase
-from openstf.pipeline.create_forecast import generate_forecast_datetime_range, generate_inputdata_datetime_range
-from openstf.validation.validation import validate, clean, is_data_sufficient
+from openstf.validation.validation import validate, is_data_sufficient, find_nonzero_flatliner
 from openstf.feature_engineering.feature_applicator import OperationalPredictFeatureApplicator
 from openstf.model.confidence_interval_applicator import ConfidenceIntervalApplicator
+from openstf.preprocessing import preprocessing
+
 MODEL_LOCATION = Path('.')
 
-def predict_pipeline(pj):
+def predict_pipeline(pj, input_data):
 
```

Comment:
Probably a good idea to add a docstring here. The pipeline also does not seem to return anything? I guess it should return the forecast right?

Reply:
Perhaps in the future we will have clear data models what a `pj` and `input_data` contains. But for the time being it should probably be documented when used. Without the documentation it is unknown which datatype and structure there arguments need to have.

Reply:
I added a minimal docstring for the predict pipeline. It now returns the forecast.

Reply:
Looks good.